storage/innobase/handler/ha_innodb.cc

```diff
@@ -21686,6 +21720,12 @@ static MYSQL_SYSVAR_BOOL(sync_debug, srv_sync_debug,
   PLUGIN_VAR_OPCMDARG | PLUGIN_VAR_READONLY,
   "Enable the sync debug checks",
   NULL, NULL, FALSE);
+
+static MYSQL_SYSVAR_ULONG(master_encrypt_debug,
+  srv_master_encrypt_debug,
+  PLUGIN_VAR_OPCMDARG,
+  "Pause in the middle of the enabling of temporary tablespace encryption.",
```

Comment:
Could you please describe 0, 1, 2 values here to make it easier to use for any future testcases?

Reply:
done